src/App/docutils/ComponentPreview/EditableComponentPreview.js

```diff
@@ -0,0 +1,275 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import {
+    SandpackProvider,
+    SandpackCodeEditor,
+    SandpackPreview,
+} from "@codesandbox/sandpack-react";
+import { ShowCasePanelAdvanced } from "./components/ShowCasePanelAdvanced";
+import { useCodeParser } from "./hooksUtils/useCodeParser";
+import { resetOptions } from "./hooksUtils/optionsUtils";
+import { useJsSandboxInitCode } from "./hooksUtils/useSandboxInitCode";
+import packageJson from "~/package";
+
+// TODO: should the preview of showCasePanel & the one of advancedShowCasePanel use the same component ?
+// in a way it is using the same SndpackPreview, so it would keep them in sync (e.g. add additional button "copy to clipboard", etc)
+// on the other hand the whole container around is different (for some reason)
```

Comment:
Before merge, delete these comments, or change them to be informational 💡

Reply:
👍 done 🧹